patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
++        for (TickListChunk.a<T> ticklistchunk_a : this.a) {
+             ticklist.a(ticklistchunk_a.a, ticklistchunk_a.d, ticklistchunk_a.b, ticklistchunk_a.c);
+-        });
++        }
+     }
+ 
+     static class a<T> {
+diff --git a/src/main/java/net/minecraft/server/VillagePlaceSection.java b/src/main/java/net/minecraft/server/VillagePlaceSection.java
+index 3aaba7fcaf83a491b9da60d156b7f2379f5a6aeb..5151876951d0089fa2ae229c7fee6e418fea3636 100644
+--- a/src/main/java/net/minecraft/server/VillagePlaceSection.java
++++ b/src/main/java/net/minecraft/server/VillagePlaceSection.java
+@@ -52,7 +52,9 @@ public class VillagePlaceSection {
+         this.c = Maps.newHashMap();
+         this.d = runnable;
+         this.e = flag;
+-        list.forEach(this::a);
++        for (VillagePlaceRecord villagePlaceRecord : list) {
++            a(villagePlaceRecord);
++        }
+     }
+ 
+     public Stream<VillagePlaceRecord> a(Predicate<VillagePlaceType> predicate, VillagePlace.Occupancy villageplace_occupancy) {
+diff --git a/src/main/java/net/minecraft/server/VillagePlaceType.java b/src/main/java/net/minecraft/server/VillagePlaceType.java
+index 1fb87bf395ced15bc3ef383dae46a42c5e4d2547..5be0e3485bb7a398f4abbaf1a3b6fc57e827aa8e 100644
+--- a/src/main/java/net/minecraft/server/VillagePlaceType.java
++++ b/src/main/java/net/minecraft/server/VillagePlaceType.java
+@@ -115,13 +115,13 @@ public class VillagePlaceType {
+     }
+ 
+     private static VillagePlaceType a(VillagePlaceType villageplacetype) {
+-        villageplacetype.C.forEach((iblockdata) -> {
+-            VillagePlaceType villageplacetype1 = (VillagePlaceType) VillagePlaceType.A.put(iblockdata, villageplacetype);
++        for (IBlockData iblockdata : villageplacetype.C) {
++            VillagePlaceType villageplacetype1 = VillagePlaceType.A.put(iblockdata, villageplacetype);
+ 
+             if (villageplacetype1 != null) {
+-                throw (IllegalStateException) SystemUtils.c(new IllegalStateException(String.format("%s is defined in too many tags", iblockdata)));
++                throw SystemUtils.c(new IllegalStateException(String.format("%s is defined in too many tags", iblockdata)));
+             }
+-        });
++        }
+         return villageplacetype;
+     }
+ 
+diff --git a/src/main/java/net/minecraft/server/WeightedList.java b/src/main/java/net/minecraft/server/WeightedList.java
+index 5d9d58411f2fad9d5da703f964d269b4a7c2b205..0879d6f82803e7da963b029eb1ff6c190ab02b8a 100644
+--- a/src/main/java/net/minecraft/server/WeightedList.java
++++ b/src/main/java/net/minecraft/server/WeightedList.java
+@@ -50,7 +50,9 @@ public class WeightedList<U> {
+     public WeightedList<U> a(Random random) {
+         // Paper start - make concurrent safe, work off a clone of the list
+         List<WeightedList.a<U>> list = isUnsafe ? new java.util.ArrayList<WeightedList.a<U>>(this.list) : this.list;
+-        list.forEach((weightedlist_a) -> weightedlist_a.a(random.nextFloat()));
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/ArrayList.java#l1226